modules/distribution/src/repository/resources/dbscripts/apimgt/postgresql.sql

```diff
@@ -84,18 +85,19 @@ CREATE INDEX IDX_TC ON IDN_OAUTH2_ACCESS_TOKEN(TIME_CREATED);
 
 DROP TABLE IF EXISTS IDN_OAUTH2_AUTHORIZATION_CODE;
 CREATE TABLE IDN_OAUTH2_AUTHORIZATION_CODE (
+            CODE_ID VARCHAR (255),
             AUTHORIZATION_CODE VARCHAR(512),
             CONSUMER_KEY_ID INTEGER,
-                CALLBACK_URL VARCHAR(1024),
+	          CALLBACK_URL VARCHAR(1024),
             SCOPE VARCHAR(2048),
             AUTHZ_USER VARCHAR (100),
             TENANT_ID INTEGER,
             USER_DOMAIN VARCHAR(50),
-                TIME_CREATED TIMESTAMP,
-                VALIDITY_PERIOD BIGINT,
+	          TIME_CREATED TIMESTAMP,
+	          VALIDITY_PERIOD BIGINT,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04